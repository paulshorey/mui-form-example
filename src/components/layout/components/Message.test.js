import React from 'react';
import renderer from 'react-test-renderer';

// shallow + redux
import { shallow } from 'index.testing';
import { Message } from './Message';

it('renders correctly', () => {
	const tree = renderer
		.create(shallow(<Message message={{ title: 'Hello!', opened: true }} />))
		.toJSON();
	expect(tree).toMatchSnapshot();
});
