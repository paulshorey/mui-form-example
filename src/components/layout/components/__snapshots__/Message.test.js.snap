// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="testing!"
>
  <div
    className="sc-bdVaJa LvSIm"
  >
    <div
      className="messages_container"
    >
      <div
        className="messages_content"
      >
        <div
          className="message opened"
          onMouseEnter={[Function]}
        >
          <div
            className="message_content"
          >
            <div
              className="messageX"
              onClick={[Function]}
            >
              <span
                className="fontIcon icon-cross"
                title="hide message"
              />
            </div>
            <div
              className="message_text messageTitle"
            >
              <div>
                Hello!
                 
                
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
